migtests/tests/pg-dependent-ddls/validate

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python3
+
+import yb
+
+def main():
+	yb.run_checks(migration_completed_checks)
+
+
+#=============================================================================
+
+def migration_completed_checks(tgt):    
+	table_list = tgt.get_table_names("public")
+	print("table_list:", table_list)
+	assert len(table_list) == 2 
```

Comment:
You can strengthen the checks by explicitly comparing table names:
```
assert set(table_list) == set(["foo", "bar"])
```
Same applies to the other assertion in this function as well.

Reply:
done